### ISSUE TITLE
Alpha -> Beta

### ISSUE THREAD

**Issue by adityasaky** [gittuf/gittuf#297]
### Add a description

A couple of folks have pointed out that our (multiple) warnings about alpha status is not conducive to giving gittuf a try. We ought to consider what we want to nail down for moving to beta status. I hope to reach a nice cycle where we aren't necessarily 100% stable but we're able to test / prototype gittuf in different use cases which drive bug fixes / workflow updates etc.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by JustinCappos** [comment]
This makes sense to me.  Certainly, we should be using gittuf during gittuf development for this to happen.  :)

**Comment by adityasaky** [comment]
yes! we need to start using gittuf on this repository ASAP. I added some immediate blockers here: https://github.com/gittuf/gittuf/issues/230#issuecomment-1939528529

**Comment by adityasaky** [comment]
@JustinCappos @patzielinski @wlynch @neilnaveen @reza-curtmola: We've done a lot of work on the plumbing with policy metadata updates and so on. Now, we have a rough sketch for how to handle metadata version bumps (v0.2 of the policy metadata recently added support for "persons", for example), both for policy and attestation predicates. Do we have thoughts on what's blocking a "beta" tag?

**Comment by JustinCappos** [comment]
I think that we should move to beta or even a 1.0 release.   We've gotten some (positive) operational experience, so let's start to let people know they should consider the tooling ready for use.

**Comment by patzielinski** [comment]
Closing as gittuf is in beta as of `v0.10.0`!
